client/my-sites/checkout/composite-checkout/hooks/use-get-thank-you-url/index.tsx

```diff
@@ -62,7 +62,12 @@ export default function useGetThankYouUrl( {
 		const url = getThankYouPageUrl( getThankYouPageUrlArguments );
 		debug( 'getThankYouUrl returned', url );
 
-		return url;
+		try {
+			return encodeURIComponent( url );
```

Comment:
Interesting! I'm surprised this wasn't broken for more flows. What is the full Copy Site thank you URL, and what is it on other flows?

Reply:
I searched all other flows inside `/stepper/declarative-flow`. There are no other URLs that redirect to the checkout that have `&` in them. I think this is the reason this issue wasn't apparent till now.

The `redirectTo` copy site URL is `/setup/copy-site/automated-copy?sourceSlug=<site slug>&siteSlug=<site slug>`